packages/terra-graphs-docs/package.json

```diff
@@ -40,6 +39,5 @@
     "react": "^16.8.5",
     "react-dom": "^16.8.5"
   },
-  "scripts": {
-  }
+  "scripts": {}
```

Comment:
Should this package include the `compile`, `lint`, and `precompile` scripts similar to other [terra docs packages](https://github.com/cerner/terra-core/blob/%40cerner/terra-core-docs%401.12.0/packages/terra-core-docs/package.json#L33-L35) and make sure these script can run successfully?